docs/download.md

```diff
@@ -190,28 +190,32 @@ You can bypass this by clicking "More info" then "Run anyway".
 ## Manual download
 
 Binaries are built from a number of different branches and PRs but you should
-stick to the **master** branch releases for the most stable ones unless you know
+stick to the **`master`** branch releases for the most stable ones unless you know
```

Comment:
```suggestion
stick to the `master` branch releases for the most stable ones unless you know
```
I'd probably remove the **strong** from this as I think converting it to inline code is enough to differentiate it.